src/transformers/integrations.py

```diff
@@ -27,6 +27,7 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Dict, Optional
 
+import importlib_metadata
```

Comment:
This should be imported from `utils.versions` as this package has different names depending on Python version (this is why one of the CI check is currently failing).

Reply:
This still needs to be addressed otherwise this code won't work in Python 3.7